pandas/core/generic.py

```diff
@@ -2952,7 +2952,12 @@ def copy(self, deep=True):
         Parameters
         ----------
         deep : boolean or string, default True
-            Make a deep copy, i.e. also copy data
+            Make a deep copy, i.e. also copy data.
```

Comment:
change this to something like:

deep also copy the indexes (its not the data that addtionally gets copied; a shallow copy does this).